api/utilities/preload_models.py

```
@@ -0,0 +1,13 @@
+from transformers import AutoTokenizer, AutoModelForSequenceClassification, pipeline, AutoModelForTokenClassification
```

Ruff (0.11.9)

`transformers.pipeline` imported but unused

Remove unused import

(F401)